### ISSUE TITLE
The result of query appears blank.

### ISSUE THREAD

**Issue by Cywair** [xiaoyifang/goldendict#52]
![image](https://user-images.githubusercontent.com/23193761/163904517-e12677e7-21a4-4490-abe9-a88138c039e9.png)
My OS is Manjaro KDE Plasma 5.24.4.
Whatever I search, the result is unable to load with a busy cursor appearing.

**Comment by xiaoyifang** [comment]
you build it from source?
any logs?

**Comment by Cywair** [comment]
> you build it from source? any logs?
I used the AppImage released on April 17th. I am sorry that I don't know how to get logs.
I built it myself just now and everything seemed to work, except when I try to use shortcut to show the window, the file tab will be opened unexpectedly.

**Comment by Cywair** [comment]
> you build it from source? any logs?

I used the AppImage released on April 17th. I am sorry that I don't know how to get logs.
I built it myself just now and everything seemed to work, except when I try to use shortcut to show the window, the file tab will be opened unexpectedly.

**Comment by xiaoyifang** [comment]
AppImage does not seem work well on linux platforms.
Recommend to  build from source on Linux platforms.



> when I try to use shortcut to show the window, the file tab will be opened unexpectedly.

which shortcut ,what is the file tab?

**Comment by Cywair** [comment]
![image](https://user-images.githubusercontent.com/23193761/164340797-cb92557b-3385-480c-a631-181a9da93d05.png)
This one.
Every time I press this hotkey, it shows the window with file tab opened, just like when you also press ALT+F to open it. I also changed the hotkey, but it didn't fix the issue.
![image](https://user-images.githubusercontent.com/23193761/164341112-1ab844aa-988e-4723-9743-5d5e056b3b08.png)
I also tried to run the appimage with --log-to-file, but I am unable to understand it, so I'll stick it below.

> Load done
getResource: gdlookup://localhost?blank=1
scheme: gdlookup
host: localhost
getResource: gdlookup://localhost?blank=1
scheme: gdlookup
host: localhost
getResource: gdlookup://localhost?word=Welcome!&group=4294967295
scheme: gdlookup
host: localhost
In-place finish.
====reading 2099 bytes
getResource: gdlookup://localhost?word=test&group=4294967294
scheme: gdlookup
host: localhost
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one finished.
erasing..
erase done..
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one not finished.
some body finished
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
one finished.
erasing..
erase done..
====reading 65536 bytes

As to the one I built, I found that pressing the hotkey won't trigger logging. I think it's no use to stick the log since there's nothing useful in it.
